docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | Arbitrary string (acts as a token) | Triggers reconciliation of the resource when the value changes. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the current `Freight`. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress `Freight` verification. |
```

Comment:
For verification ID, we probably need to tell users where they can find that. "JSON object (VerificationRequest)" doesn't really tell users much of anything. Even I don't know what the structure of that JSON should look like without first digging through the code.

Since these two rows probably require snippets of `StageStatus` represented as YAML in order to be properly informative, it makes me question whether a table is the best format for this information. You can try to make it look nice, but you may need to resort to making "Annotations" and "Labels" level 2 headings (i.e. `##`) and annotation/label keys level 3 (i.e. `###`).